utils/activityLog.js

```diff
@@ -38,8 +38,10 @@ module.exports = {
    * Convert activity log object to String format and print.
    * @param  {Object} logObject - Log Object
    * @param  {string} logtype - worker, api or realtime
+   * @param  {string} logLevel - Log Level info, warn, error
+   * verbose, debug, silly. Default log info.
    */
-  printActivityLogString(logObject, logtype) {
+  printActivityLogString(logObject, logtype, logLevel='info') {
```

Comment:
I modified this function so that we can use for any kind of log. Earlier it is only for info logs.